modin/pandas/test/dataframe/test_udf.py

```diff
@@ -100,6 +100,13 @@ def test_aggregate_error_checking():
         modin_df.aggregate("NOT_EXISTS")
 
 
+@pytest.mark.xfail(
```

Comment:
1. This is expression doesn't skip this test for the `BaseOnPython` backend, so the front-end logic of `apply` is still being tested.
2. This bug became visible now because we changed the "target" exception here from `TypeError` to the `KeyError` and now the apply's buggy behavior does not match for the test

Reply:
ok